deployment/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.asc
+birt.zip
```

Comment:
Should `bin.jar` also be included in this list?

Reply:
The bin.jar in the birt-framework directory is in the repo as a placeholder, since upload to OSSRH required a binary jar file as the main jar.  It wouldn't accept just a classifier=zip artefact on it's own :(

Reply:
:confused: Maven and Gradle upload artifacts to OSSRH independent of source control (artifacts usually aren't checked in).
Does Eclipse' upload process work different?

Reply:
If you create a normal Gradle project in Eclipse then it would create normal jars etc, and you can get your dependencies via Maven etc...
The problem is that these jars are actually OSGI plugins, and the dependencies are fetched via "P2" (you can't hope the retrieve them from Maven), therefore I had serious trouble in getting a Gradle script to build the Eclipse plugins and automatically push them to Maven. (Hence the manual process).  It might be possible, using a number of Gradle plugins that I tried and failed with, but I had already spent way too long trying to get those to work. (See the aborted feature/build-process branch)